src/components/BlubberFormFactory.js

```diff
@@ -811,11 +988,11 @@ Vue.mixin( {
 				GeneratedFields.push( Mutable );
 			}
 
-			if ( Object.isEmpty( GeneratedFields ) === false ) {
+			if ( ObjectHelper.isEmpty( GeneratedFields ) === false ) {
```

Comment:
I'm not a fan of checking `=== true/false` for functions that are guaranteed to return booleans, but maybe this is a matter of taste.